src/get/getReleaseInfo.js

```diff
@@ -28,6 +28,10 @@ export const getReleaseInfo = async (version, cacheDir, manifestUrl) => {
     log.debug("Convert manifest data into JSON");
     let manifestJson = JSON.parse(manifestData);
     log.debug(`Search for ${version} specific release data`);
+    if (version === "latest" || version === "lts") {
+      version = manifestData[version].slice(1);
```

Comment:
Reassigning function parameters! 😢 

I don't think "lts" makes sense for anything NW.js-related. Unfortunately, there aren't any LTS builds available and they don't even match the Node.js LTS builds. So, supporting an "lts" version is misleading.

I also noticed that you mentioned a "stable" version being supported in the PR comments. This would be the same as "lts", where it simply doesn't exist.

Reply:
Another thought, can `latest` become the default if users didn't specify a version?

Reply:
LTS in this case refers to NW.js v0.14.7. I believe this is the last version to support Windows XP. The current NW release flow matches the latest Node version with the latest Chromium version.

p.s.: I am planning to [do custom builds](https://github.com/tharatau/nw/issues/1) but setting that up will take some time.

Reply:
I don't agree with calling 0.14.7 "LTS", but that's your call. That version is simply not "long term support". It's not supported anymore and it doesn't work like any remotely-modern version. If anything, "legacy" would be a more-fitting name.

Either way, what are you considering "stable"? There is no branch that's considered "stable", there's only "latest".

Reply:
I just looked at `https://nwjs.io/versions` and see where you're getting the lts/stable from. Those really don't make sense for how Roger releases NW.js, but use whatever you want!

Reply:
> I don't agree with calling 0.14.7 "LTS"

I agree but I think that is an upstream issue.

> If anything, "legacy" would be a more-fitting name.

I'll leave lts and stable in for now. I'll open an issue upstream.